tests/integration/targets/ec2_instance_termination_protection/tasks/main.yml

```diff
@@ -52,23 +52,17 @@
           - create_instance_results.spec.DisableApiTermination
 
     - name: Get info on termination protection
-      ansible.builtin.command: aws ec2 describe-instance-attribute --attribute disableApiTermination --instance-id {{ instance_id }}
-      environment:
-        AWS_ACCESS_KEY_ID: "{{ aws_access_key }}"
-        AWS_SECRET_ACCESS_KEY: "{{ aws_secret_key }}"
-        AWS_SESSION_TOKEN: "{{ security_token | default('') }}"
-        AWS_DEFAULT_REGION: "{{ aws_region }}"
-      register: instance_termination_check
-      changed_when: false
-
-    - name: Convert it to an object
-      ansible.builtin.set_fact:
-        instance_termination_status: "{{ instance_termination_check.stdout | from_json }}"
+      amazon.aws.ec2_instance_info:
+        instance_ids:
+          - "{{ instance_id }}"
+        include_attributes:
+          - disableApiTermination
+      register: instance_info
 
     - name: Assert termination protection status did not change in check_mode
       ansible.builtin.assert:
         that:
-          - instance_termination_status.DisableApiTermination.Value == true
+          - instance_info.instances.0.attributes.disable_api_termination.value
```

Comment:
Could we retain the `== true` and `== false` part of these checks for clarity? Just noting the comments [here](https://github.com/nitzmahone/ansible-documentation/blob/core_data_tagging/docs/docsitec/rst/porting_guides/porting_guide_core_2.19.rst#broken-conditionals) around implicit truthiness checks and thinking it's better to be explicit where we can.